xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -48,19 +58,16 @@ static Builder newBuilder() {
     }
 
     static final class Builder {
-      private ConfigUpdate base;
-
-      private Builder() {
-        base = new ConfigUpdate();
-      }
+      private String clusterName;
```

Comment:
You still need private constructor to prevent default constructor from being called outside of the class, in favor of `ConfigUpdate.newBuilder()`.

Reply:
Done.